src/code_replica.jl

```diff
@@ -25,12 +25,13 @@ function gen_code_replica!(
     most_late_sample_shift  = correlator_sample_shifts[1]
     total_samples = num_samples + most_early_sample_shift - most_late_sample_shift
```

Comment:
Commenting only on variable naming: wouldn't it be better to name the variables `latest_sample_shift`and `earliest_sample_shift`?

Reply:
Actually, this would lead to shorter variable names, but is probably out of scope here, what do you think @zsoerenm?

Reply:
Yes, I support this change. Would you mind to create a pull request @ozmaden ?